api/scripts/certification/copy-data-calibrations.js

```diff
@@ -0,0 +1,73 @@
+import { knex as datamartKnex } from '../../datamart/knex-database-connection.js';
+import { knex } from '../../db/knex-database-connection.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+class CopyDataCalibrations extends Script {
+  constructor() {
+    super({
+      description: 'Copy data about calibrations from datawarehouse to API',
+      permanent: false,
+    });
+  }
+
+  async handle({ _, logger }) {
```

Comment:
```suggestion
  async handle({ logger }) {
```
Pas besoin de ce paramètre s'il n'est pas utilisé.